read-the-docs-2016-stats.rst

```diff
@@ -0,0 +1,141 @@
+
+Page Views
+----------
+
+Our stats:
+
+* 252 Million Page Views (+48%)
+* 56 Million Unique Visitors (+47%)
+
+.. From Google Analytics
+
+Page view growth has been steady again this year.
+This is particularly impressive given the scale that we're operating at.
+I'm quite happy to see folks using Read the Docs more and more,
+as well as seeing many other projects using our theme or other concepts that we've pioneered in the general software ecosystem.
+
+We remain one of the `largest sites`_ on the internet.
+
+.. _largest sites: http://www.alexa.com/siteinfo/readthedocs.io
+
+Site Stats
+----------
+
+The stats, in total numbers:
+
+* 53673 projects (+91%)
+* 64585 users (+65%)
+
+Read the Docs has some high profile projects that push a lot of traffic.
+There are however thousands of smaller libraries and projects that fill out that full range of documentation that we host.
+
+We are happy to host documentation for all open source projects,
+and are glad the community finds the service useful.
+
+Funding
+-------
+
+Our hosting costs continue to be sponsored by `Rackspace`_,
+which is fantastically generous of them.
+
+Our largest initiative this year has been our attempt at building `Ethical Advertising`_ into our site.
+The project has been well received,
+which we have been grateful for.
+Funding is a fundamental part of sustainability,
+and we're glad people are supportive of our efforts to get there.
+
+The best outcome of our sustainability efforts is that we're going to be able to hire someone part time to handle support for our users.
+This has always been somewhere that I've felt we have not done our best work,
+and I'm excited we will be able to give it more sustained work.
+
+Our other efforts at sustainability are also progressing. Our `commercial hosting service`_ is starting to slowly build revenue from users who want private documentation.
+We hope to continue to build that into another source of revenue,
+which can help support development of the code base.
+
+.. _commercial hosting service: https://readthedocs.com/
+.. _Ethical Advertising: http://docs.readthedocs.io/en/latest/ethical-advertising.html
+.. _Sustainabilty Campaign: https://readthedocs.org/sustainability/
+
+Conclusion
+----------
+
+2016 has been another year of mixed results.
+
+We are continuing to see more users and folks who are reading documentation on our platform.
+We think the software industry is caring more and more about documentation by the year,
+and we're helping to play a role.
+
+The downside is that as a community project,
+we still haven't getting much contribution.
+I think we're pretty firmly set on the path of having paid contributors,
+after years of trying to cultivate a volunteer community to help support the project.
+I have mixed feelings about this,
+but knowing that there is a sustainable future for the project is the most important thing in my book.
```

Comment:
There's a similar blame-y tone here. We should own this as our problem, even if we're only part to blame.